docs/ref/models/querysets.txt

```diff
@@ -1707,9 +1707,12 @@ one, doing so will result in an error.
 
 .. method:: only(*fields)
 
-The ``only()`` method is more or less the opposite of :meth:`defer()`. You call
-it with the fields that should *not* be deferred when retrieving a model.  If
-you have a model where almost all the fields need to be deferred, using
+
+The ``only()`` method is essentially the opposite of :meth:`defer()`. Only the 
```

Comment:
```suggestion
The ``only()`` method is essentially the opposite of :meth:`defer`. Only the 
```

I thought we don't need the `()` as it's added by sphinx? 🤔

Reply:
Yes, `()` are unnecessary with `:func:` and `:meth:`.

Reply:
(I believe) I've merged the suggestions made by @smithdc1 into this PR. Thanks for the input!